packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -166,6 +168,9 @@ function Preview({
     if (type === "Leave") {
       return;
     }
+    if (type === "RightButtonDown") {
+      sendTelemetry("inspector:right-button-down", {});
```

Comment:
```suggestion
      sendTelemetry("inspector:show-hierarchy", {});
```
maybe its better to describe what is does and not what is it